src/features/settings/rpc/getMembershipsList.tsx

```diff
@@ -0,0 +1,38 @@
+import { z } from 'zod';
+
+import IApiClient from 'core/api/client/IApiClient';
+import { makeRPCDef } from 'core/rpc/types';
+import { ZetkinMembership, ZetkinOfficial } from 'utils/types/zetkin';
+
+const paramsSchema = z.object({
+  orgId: z.number(),
+});
+
+type Params = z.input<typeof paramsSchema>;
+type Result = ZetkinMembership[];
+
+export const getMembershipsListDef = {
+  handler: handle,
+  name: 'getMembershipsList',
```

Comment:
Because this RPC loads very specific memberships (the ones belonging to officials) I think it should be called `getOfficialMemberships` instead.